src/Magnum/DartIntegration/Test/CMakeLists.txt

```diff
@@ -42,9 +43,9 @@ endif()
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/configure.h.cmake
                ${CMAKE_CURRENT_BINARY_DIR}/configure.h)
 
-corrade_add_test(DartIntegrationDartSkeletonTest DartSkeletonTest.cpp LIBRARIES MagnumDartIntegration ${DART_LIBRARIES})
-target_include_directories(DartIntegrationDartSkeletonTest PUBLIC ${DART_INCLUDE_DIRS})
+corrade_add_test(DartIntegrationDartWorldTest DartWorldTest.cpp LIBRARIES Magnum::OpenGLTester MagnumDartIntegration ${DART_LIBRARIES})
```

Comment:
Because the test needs GL now, in order to keep things running properly on particular systems and the CIs (which don't have access to GL at the moment) there needs to be:

- a new `BUILD_GL_TESTS` CMake option (see [here](https://github.com/mosra/magnum-extras/blob/12d2310e177019ccb6b1f40cacd95817c9856df2/CMakeLists.txt#L62) for how's that done elsewhere -- search for BUILD_GL_TESTS in that file)
- this test named `DartIntegrationWorldGLTest` (so it can be easily black/whitelisted during `ctest` run)
- this test added only `if(BUILD_GL_TESTS)` (see e.g. [here](https://github.com/mosra/magnum/blob/52613a2ad94e7c6f11c7898e162f636db7d07980/src/Magnum/Test/CMakeLists.txt#L130) for an example), don't forget to split out the `set_target_properties()` command below as well

Sorry that most of the supporting structures for GL tests are not here yet -- it was just not needed (or not cared for) until now :)

Reply:
> Sorry that most of the support code for GL tests is not here yet -- it was not needed (or not cared for) until now :)

No problem! Should I also do the `-E GLTest` as [here](https://github.com/mosra/magnum-extras/blob/master/package/ci/travis-desktop.sh#L55)?

Reply:
Yes, please. I thought at least that is done already, but apparently just partially -- in particular [this line](https://github.com/mosra/magnum-integration/blob/bb743603b1910da2936c387fd59bcabfb5d67b70/package/ci/travis-desktop.sh#L74) should say `-R Dart -E GLTest`.